tealr/src/type_walker.rs

```diff
@@ -7,30 +7,62 @@ use crate::{
     Direction, TealMultiValue, TealType, TypeBody, TypeName,
 };
 
+pub(crate) fn get_meta_name(name: rlua::MetaMethod) -> &'static str {
+    match name {
+        MetaMethod::Add => "__add",
+        MetaMethod::Sub => "__su",
```

Comment:
There is a typo here, this should be "__sub"!

Reply:
that I missed that is... not good.

Good catch. Will try and fix it today :)

Reply:
No worries, just found out when generating docs, it doesn't actually mess with documentation just the .d.tl files

Reply:
fixed by #51